src/Database/DBA.php

```diff
@@ -210,9 +212,8 @@ public static function databaseName() {
 	 * @throws \Exception
 	 */
 	private static function logIndex($query) {
-		$a = \get_app();
 
-		if (!$a->getConfigVariable('system', 'db_log_index')) {
```

Comment:
How was this even working?

Reply:
No, I mean that the `App->getConfigVariable` method never existed, only `App->getConfigValue` did.